### PR TITLE
Create Verify check and trigger_next_event blocks

### PR DESCRIPTION
**Pull request by marklewin**
Also standardize variable naming and use of ES6 `=>` across all blocks

### CODE REVIEW

File: verify/cancel.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-  apiKey: NEXMO_API_KEY,
-  apiSecret: NEXMO_API_SECRET
+const Nexmo = require('nexmo');
+const nexmo = new Nexmo({
+    apiKey: NEXMO_API_KEY,
+    apiSecret: NEXMO_API_SECRET
 }, {
-  debug:true
-});
+        debug: true
```
Comment (AlexLakatos) [inline verify/cancel.js]: the indentation here seems to be off

File: verify/check.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    return;
+}
+
+const Nexmo = require('nexmo');
+const nexmo = new Nexmo({
+    apiKey: NEXMO_API_KEY,
+    apiSecret: NEXMO_API_SECRET
+}, {
+        debug: true
+    });
```
Comment (AlexLakatos) [inline verify/check.js]: indentation here seems to be off

File: verify/next-event.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    return;
+}
+
+const Nexmo = require('nexmo');
+const nexmo = new Nexmo({
+    apiKey: NEXMO_API_KEY,
+    apiSecret: NEXMO_API_SECRET
+}, {
+        debug: true
+    });
```
Comment (AlexLakatos) [inline verify/next-event.js]: indentation seems to be off here

**Review by AlexLakatos — APPROVED** [review]
Othere than the minor indentation nits, LGTM 🌸

**Comment by AlexLakatos** [comment]
I've made the formatting updates @marklewin, if you're happy with those you can merge when you want to.

**Review by mheap — APPROVED** [review]

**Pull request merged**